Applications/SimpleShot/task_runner.cpp

```diff
@@ -110,25 +121,72 @@ const std::string getcwd_() {
 
 using LayerHandle = std::shared_ptr<ml::train::Layer>;
 
+/**
+ * @brief find model name by using strcmp
+ * @param backbone name
+ * @return int to identify the backbone
+ */
+
+int find_model_name(std::string backbone) {
```

Comment:
```suggestion
int findModelName(std::string backbone) {
```

Reply:
input looks like it's okay to use `const std::string &backbone` to avoid unnecessary copy